packages/context/src/context.ts

```diff
@@ -191,6 +274,21 @@ export class Context {
     return this._mergeWithParent(bindings, parentBindings);
   }
 
+  /**
+   * Create a binding filter from key pattern
+   * @param keyPattern Binding key, wildcard, or regexp
+   */
+  static bindingKeyFilter(keyPattern?: string | RegExp) {
```

Comment:
Let's start the method name with a verb please, e.g. `createBindingKeyFilter`.

Also AFAICT, this method does not need anything from the Context class, have you considered changing it into a regular function (not a static method)?